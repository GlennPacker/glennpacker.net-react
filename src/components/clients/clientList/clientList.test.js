import React from 'react';
import { shallow } from 'enzyme';

import ClientList from './index';

describe('clientList', () => {
    let wrapper
    beforeAll(() => {
        wrapper = shallow(<ClientList />);
    })

    it('should render', () => {
        expect(wrapper).not.toBeNull();
    })

    it('should render correctly compared to snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    })
})