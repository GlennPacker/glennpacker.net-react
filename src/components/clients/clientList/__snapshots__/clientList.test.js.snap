// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`clientList should render correctly compared to snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
